iModelJsNodeAddon/MemoryLeak_Suppressions.txt

```diff
@@ -4,4 +4,5 @@ leak:*/bsitools/*
 leak:*/BentleyBuild/*
 leak:*/toolcache/*
 leak:*/node*/*
-leak:*interceptor_malloc
\ No newline at end of file
+leak:*interceptor_malloc
+leak:*imodeljs.node*
```

Comment:
Adding this will probably ignore everything in imodeljs.node. That doesn't seem desirable